db/schema.rb

```diff
@@ -16,164 +16,158 @@
   enable_extension "plpgsql"
 
   create_table "alignments", force: :cascade do |t|
-    t.integer  "outcome_id",          null: false
```

Comment:
They aren't. It's an artifact of a Rails 5.1 upgrade. I can do them separately and then rebase